HACKING.md

```diff
@@ -341,17 +341,28 @@ oc annotate secret/cosa-push-registry-secret \
 
 ### [PROD] Create fedora-messaging configuration
 
-First create the configmap:
+First create the Fedora Messaging configuration secret:
 
 ```
-oc create configmap fedora-messaging-cfg --from-file=configs/fedmsg.toml
+oc create secret generic fedora-messaging-config \
```

Comment:
Storing the config as a secret is awkward, but I don't have a better suggestion. Would be cool if the file supported inlining the cert (it doesn't), then it could just be one secret.

Reply:
yep :(